common/src/language/mod.rs

```diff
@@ -1,25 +1,36 @@
 use std::collections::HashMap;
 
-use fluent_templates::{once_cell::sync::Lazy, LanguageIdentifier, Loader};
+use fluent_templates::{
+    lazy_static::lazy_static, once_cell::sync::Lazy, LanguageIdentifier, Loader,
+};
 use unic_langid::langid;
 use warp::sync::RwLock;
 
 use crate::LOCALES;
 
 pub const US_ENGLISH: (LanguageIdentifier, &str) = (langid!("en-US"), "English (USA)");
-const BR_PORTUGUESE: (LanguageIdentifier, &str) = (langid!("pt-BR"), "Português (Brasil)");
-const PT_PORTUGUESE: (LanguageIdentifier, &str) = (langid!("pt-PT"), "Português (Portugal)");
-const MX_SPANISH: (LanguageIdentifier, &str) = (langid!("es-MX"), "Español (México)");
+
+lazy_static! {
```

Comment:
It might be worth nothing that using `Lazy` from `OnceCell` could be used over `lazy_static`, and in some cases is preferred due to the flexibility (and would help reduce the 2 different variants).

Reply:
changed it